### PR TITLE
UICAL-214 test ExceptionFieldUtils

### PR DESCRIPTION
**Pull request by zburke**
Tests for 7/10 functions. Sorry; I picked the easy ones. 

Refs [UICAL-214](https://issues.folio.org/browse/UICAL-214)

### CODE REVIEW

File: src/components/fields/ExceptionFieldUtils.test.js [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+describe('updateInnerRowState', () => {
+  it('correctly updates given row', () => {
+    const r1 = { rows: [{ foo: 'bar' }, { bat: 'baz' }] };
+    const r2 = { rows: [{ foo: 'bar' }, { bat: 'baz' }] };
+    const r3 = { rows: [{ foo: 'bar' }, { bat: 'baz' }] };
+
+    const rowStates = [r1, r2, r3];
+
+    const newState = { monkey: 'bagel' };
```
Comment (ncovercash) [inline src/components/fields/ExceptionFieldUtils.test.js:316]: 11/10 state choice 👌

File: src/components/fields/ExceptionFieldUtils.test.js [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+        ]
+      };
+
+      const response = isInnerRowConflicted(av, 1, 7);
+      expect(response).toBe(false);
+    });
+  });
+});
+
+// export type ExceptionFieldErrors = RequireExactlyOne<{
```
Comment (ncovercash) [inline src/components/fields/ExceptionFieldUtils.test.js]: Should these comments still be here?

**Review by ncovercash — CHANGES_REQUESTED** [review]
Looks good overall, one nitpick but it looks great!  Totally understand picking the easy ones -- this component is quite complex (although I tried to break it down as best I could).

The other ones also rely on Dayjs, which I'm in the process of refactoring away, so I'll probably tackle the other functions once that's gone (although most of the validation functions themselves are tested already elsewhere, so we wouldn't need much test duplication here).

File: src/components/fields/ExceptionFieldUtils.test.js (hunk 2 above)
Comment (zburke) [reply src/components/fields/ExceptionFieldUtils.test.js]: whoops; nope; sorry; thanks.

**Review by ncovercash — APPROVED** [review]

**Pull request merged**